.circleci/config.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       - yarn/setup
       - run:
           name: Test
-          command: yarn test
+          command: COMMIT_HASH_FULL=$(git rev-parse HEAD) CODECOV_TOKEN=$CODECOV_TOKEN BRANCH_NAME=$CIRCLE_BRANCH yarn test
```

Comment:
Out of curiosity why do we need the `COMMIT_HASH` again? We were able to remove this when we migrated off of bucket-assets.

Reply:
We send this to codecov to upload reports, this is the format of what we send them: 
`
{ commit: 'd41b687186251342927045b4fb91ca238d139c4c',
  branch: 'pull/3930',
  package: 'node-v3.3.0' }
`

It is also used in the urls for reports like this one: https://codecov.io/github/artsy/force/commit/d41b687186251342927045b4fb91ca238d139c4c

The var only lives in the circle test step, and is added to the `.env.test` but we do not ever need to set it anywhere.

Reply:
Sounds good - ty for explaining 👍